--- a/.vite/build/workspace-root-drop-handler-4fzIumU3.js
+++ b/.vite/build/workspace-root-drop-handler-4fzIumU3.js
@@ -40245,6 +40245,7 @@
     i = t.Hi(e.version);
   return (
     Lz({
+      enabled: !1,
       dsn: t.Vi,
       environment: rB.buildFlavor,
       release: i,
--- a/.vite/build/worker.js
+++ b/.vite/build/worker.js
@@ -74382,6 +74382,7 @@
   let n = AH(e.appVersion),
     r = e.buildFlavor !== `prod`;
   (dO({
+    enabled: !1,
     dsn: EW,
     environment: e.buildFlavor,
     release: DW(n.version),
