--- a/webview/assets/app-initial~app-main~worktree-init-v2-page~appgen-settings-page~page~appgen-page~remote-con~lblzrcac-CGLvIrB3.js
+++ b/webview/assets/app-initial~app-main~worktree-init-v2-page~appgen-settings-page~page~appgen-page~remote-con~lblzrcac-CGLvIrB3.js
@@ -13897,6 +13897,7 @@
     n = R(e.appVersion),
     r = e.buildFlavor !== `prod`;
   (Lw({
+    enabled: !1,
     beforeSend: ot,
     dsn: On,
     environment: Gw,
