--- a/webview/assets/app-initial-236e1501144c.js
+++ b/webview/assets/app-initial-236e1501144c.js
@@ -172745,11 +172745,16 @@
   useTransitions: i,
 }) {
   let a = dT.useRef();
-  a.current ??= F_n({ initialEntries: n, initialIndex: r, v5Compat: !0 });
+  a.current ??= F_n({
+    initialEntries: n ?? [window.__ELECTRON_SHIM__.initialRoute],
+    initialIndex: r,
+    v5Compat: !0,
+  });
   let o = a.current,
     [s, c] = dT.useState({ action: o.action, location: o.location }),
     l = dT.useCallback(
       (e) => {
+        window.__ELECTRON_SHIM__.onMemoryNavigationChanged(e);
         i === !1 ? c(e) : dT.startTransition(() => c(e));
       },
       [i],
@@ -200768,7 +200773,7 @@
   return RT(f0n, void 0) !== void 0;
 }
 function s0n() {
-  return !0;
+  return window.__ELECTRON_SHIM__.initialSidebarState;
 }
 function c0n() {
   return RT(f0n, !1);
