--- a/webview/assets/app-initial~app-main~pull-request-code-review~onboarding-page~hotkey-window-thread-page~cha~b76hmflu-y0KJWbm3.js
+++ b/webview/assets/app-initial~app-main~pull-request-code-review~onboarding-page~hotkey-window-thread-page~cha~b76hmflu-y0KJWbm3.js
@@ -140779,11 +140779,16 @@
   unstable_useTransitions: i,
 }) {
   let a = AM.useRef();
-  a.current ??= gJt({ initialEntries: n, initialIndex: r, v5Compat: !0 });
+  a.current ??= gJt({
+    initialEntries: n ?? [window.__ELECTRON_SHIM__.initialRoute],
+    initialIndex: r,
+    v5Compat: !0,
+  });
   let o = a.current,
     [s, c] = AM.useState({ action: o.action, location: o.location }),
     l = AM.useCallback(
       (e) => {
+        window.__ELECTRON_SHIM__.onMemoryNavigationChanged(e);
         i === !1 ? c(e) : AM.startTransition(() => c(e));
       },
       [i],
@@ -162985,7 +162990,7 @@
       spn(),
       (_pn = `app-shell-bottom-panel-launcher-visible`),
       (vpn = 100),
-      (gR = ga(G, !0)),
+      (gR = ga(G, window.__ELECTRON_SHIM__.initialSidebarState)),
       (ypn = ga(G, !0)),
       (_R = ga(G, !1)),
       (vR = ga(G, !1, {
@@ -163028,7 +163033,7 @@
           };
         },
       })),
-      (xpn = ga(G, () => new vp(1))),
+      (xpn = ga(G, () => new vp(window.__ELECTRON_SHIM__.initialSidebarState))),
       (Spn = ga(G, npn)),
       (Cpn = K_(_pn, !0)),
       (yR = ga(iM, !1)),
