--- a/webview/assets/app-initial~app-main~settings-page-xBxCLW__.js
+++ b/webview/assets/app-initial~app-main~settings-page-xBxCLW__.js
@@ -1,4 +1,4 @@
-import { n as e } from "./rolldown-runtime-Czos8NxU.js";
+import { n as e, s as Vs } from "./rolldown-runtime-Czos8NxU.js";
 import {
   Ah as t,
   Cu as n,
@@ -33,6 +33,7 @@
   iU as D,
   ju as ie,
   ku as O,
+  lZ as Ur,
   mH as k,
   nw as A,
   oU as ae,
@@ -67,11 +68,17 @@
   f(`toggleSidebar`, i);
   let o;
   (e[2] === n ? (o = e[3]) : ((o = [n]), (e[2] = n), (e[3] = o)),
-    b(`toggle-sidebar`, i, o));
+    b(`toggle-sidebar`, i, o),
+    Wr.useEffect(() => {
+      window.__ELECTRON_SHIM__.closeSidebar = () => {
+        t(n, !1);
+      };
+    }, [n]));
 }
 var U,
+  Wr,
   W = e(() => {
-    ((U = E()), s(), l(), F(), r(), y());
+    ((U = E()), (Wr = Vs(Ur(), 1)), s(), l(), F(), r(), y());
   }),
   G,
   ce = e(() => {
