--- a/webview/assets/app-initial~app-main~worktree-init-v2-page~remote-conversation-page~onboarding-page~hotkey-~ke3yc5wu-BLQiF1Gs.js
+++ b/webview/assets/app-initial~app-main~worktree-init-v2-page~remote-conversation-page~onboarding-page~hotkey-~ke3yc5wu-BLQiF1Gs.js
@@ -87229,11 +87229,16 @@
   unstable_useTransitions: i,
 }) {
   let a = IB.useRef();
-  a.current ??= smt({ initialEntries: n, initialIndex: r, v5Compat: !0 });
+  a.current ??= smt({
+    initialEntries: n ?? [window.__ELECTRON_SHIM__.initialRoute],
+    initialIndex: r,
+    v5Compat: !0,
+  });
   let o = a.current,
     [s, c] = IB.useState({ action: o.action, location: o.location }),
     l = IB.useCallback(
       (e) => {
+        window.__ELECTRON_SHIM__.onMemoryNavigationChanged(e);
         i === !1 ? c(e) : IB.startTransition(() => c(e));
       },
       [i],
@@ -110537,7 +110542,7 @@
       szt(),
       (NX = `app-shell-bottom-panel-launcher-visible`),
       (mzt = 100),
-      (PX = Ji(Z, !0)),
+      (PX = Ji(Z, window.__ELECTRON_SHIM__.initialSidebarState)),
       (FX = Ji(Z, !1)),
       (IX = Ji(Z, !1)),
       (LX = Ji(Z, !1)),
@@ -110563,7 +110568,7 @@
           };
         },
       })),
-      (zX = Ji(Z, () => new lb(1))),
+      (zX = Ji(Z, () => new lb(window.__ELECTRON_SHIM__.initialSidebarState))),
       (_zt = Ji(Z, izt)),
       (BX = gD(NX, !0)),
       (VX = Ji(bV, !1)),
