--- a/webview/assets/app-initial~app-main~pull-request-code-review~onboarding-page~hotkey-window-thread-page~cha~b76hmflu-y0KJWbm3.js
+++ b/webview/assets/app-initial~app-main~pull-request-code-review~onboarding-page~hotkey-window-thread-page~cha~b76hmflu-y0KJWbm3.js
@@ -253935,7 +253935,7 @@
         })));
   });
 function qbr(e) {
-  return `${Xbr}${Ybr(e)}`;
+  return Ybr(e);
 }
 function Jbr(e) {
   return Ybr(e);
